Champions/Global/SummonerHeal.cs

```diff
@@ -39,41 +39,32 @@ public void OnFinishCasting(Champion owner, Spell spell, AttackableUnit target)
 
             if (mostWoundedAlliedChampion != null)
             {
-                newHealth = mostWoundedAlliedChampion.Stats.CurrentHealth + 75 + owner.Stats.Level * 15;
+                healthGain = 75 + (owner.Stats.Level * 15);
```

Comment:
I think this healing routine can be extracted into its own function, so this could be like this:
```cs
public void OnFinishCasting()
{
    // ...
    DoHeal(mostWoundedAlliedChampion);
    DoHeal(owner);
}

public void DoHeal(Champion target)
{
    // heal
}
```